src/Core/src/Platform/iOS/MauiLabel.cs

```diff
@@ -0,0 +1,26 @@
+﻿using CoreGraphics;
+using UIKit;
+
+namespace Microsoft.Maui.Platform.iOS
+{
+	public class MauiLabel : UILabel
+	{
+		public UIEdgeInsets TextInsets { get; set; }
+
+		public MauiLabel(CGRect frame) : base(frame)
+		{
+		}
+
+		public MauiLabel()
+		{
+		}
+
+		public override void DrawText(CGRect rect) => base.DrawText(TextInsets.InsetRect(rect));
+
+		public override CGSize SizeThatFits(CGSize size) => AddInsets(base.SizeThatFits(size));
+
+		CGSize AddInsets(CGSize size) => new CGSize(
+			width: size.Width + TextInsets.Left + TextInsets.Right,
+			height: size.Height + TextInsets.Top + TextInsets.Bottom);
+	}
+}
```

Comment:
For this particular property, we needed to extend the UILabel class with some custom logic because Padding doesn't exist natively. This class was ported from the Forms iOS platform project.